BurnerB/FreeMentors-V2
# #168400633 Mentor Accept deny session

#### What does this PR do?
Allows a mentor to accept or reject a session
#### Description of Task to be completed?
* User should be able to accept or deny based on the url
* Tests should pass
* The app  should work with a db
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ft-DB-accept-deny-session-168400633
* Run npm --init to install all dependancies.
* Run command npm test on your terminal
* Tests should pass
* Open postman and PATCH `/sessions/:sessionId/accept` or `/sessions/:sessionId/decline`
* 
 * Should recieve response:
 ```
 {
   "status": 200,
    "data":
        {
        sessionId:1
        mentorId:1
        menteeId:1
        questions:"a question"
        menteeEmail:"email"
        status:"accepted or declined"
            
        },
}
 ```
#### What are the relevant pivotal tracker stories?
[#168400633](https://www.pivotaltracker.com/story/show/168400633)
#### Screenshots (if appropriate) 
![session accept](https://user-images.githubusercontent.com/45785786/64688552-3ec7ae00-d495-11e9-9483-9ef1e0bcdf9c.png)
![already accpted](https://user-images.githubusercontent.com/45785786/64688555-412a0800-d495-11e9-896a-6751fb839d17.png)
